src/test/java/systemtests/DeleteCommandSystemTest.java

```diff
@@ -37,6 +36,7 @@ public void delete() {
         String expectedResultMessage = String.format(MESSAGE_DELETE_PERSON_SUCCESS, deletedPerson);
         assertCommandSuccess(command, expectedModel, expectedResultMessage);
 
+
```

Comment:
Be careful of unnecessary or accidental changes here as it can cause unnecessary merge conflict.